src/components/table/JadeTable.jsx

```diff
@@ -126,25 +131,34 @@ export class JadeTable extends React.PureComponent {
   };
 
   handleArrayValues = (value, column) => {
+    const returnValue = [];
+    if (column.mode === COLUMN_MODES.REPEATED) {
+      returnValue.push(<span key="start">[</span>);
+    }
+    returnValue.push(
+      ...value.flatMap((v, i) => [
+        v,
```

Comment:
what if the value in the array is null?  I think you want
```suggestion
        _.isNull(v) ? this.handleNullValue(v) : v,
```